packages/esm-patient-registration-app/src/root.component.tsx

```diff
@@ -46,6 +46,10 @@ export default function Root({ savePatientForm, isOffline }: RootProps) {
                 path=""
                 element={<PatientRegistration savePatientForm={savePatientForm} isOffline={isOffline} />}
               />
+            </Routes>
+          </BrowserRouter>
+          <BrowserRouter basename={`${window['getOpenmrsSpaBase']()}`}>
```

Comment:
It would be better to just combine this into a single BrowserRouter component. Also, the `path` should be `patient/:patientUuid/edit`...

Reply:
Thanks for your feedback, should be done now.